README.md

```diff
@@ -76,19 +76,19 @@ The files contained in this repository are as follows:
 
 ---
 
-#### [222-connect.nulogy.net.crt](222-connect.nulogy.net.pem) - TLS Certificate for Production Environment
+#### [224-connect.nulogy.net.crt](224-connect.nulogy.net.pem) - TLS Certificate for Production Environment
 
 - Subject: `CN=connect.nulogy.net,O=Nulogy Corporation,L=Toronto,ST=Ontario,C=CA` (Issuer: `CN=DigiCert Global G2 TLS RSA SHA256 2020 CA1,O=DigiCert Inc,C=US`)
-- Serial Number: 9034213788272798128800337420850908347 (`0x6cbed211256807796478204ef054cbb`)
-- Expires **August 2024**
+- Serial Number: 13207844988586335798285133674894363845 (`0x9efbd08dc0448356218fdb614ac8cc5`)
+- Expires **August 30 2025**
 - Contents include:
   - Certificate (connect.nulogy.net)
   - Intermediate CA Certificate (DigiCert Global G2 TLS RSA SHA256 2020 CA1)
   - Root CA Certificate ([DigiCert Global Root G2](#digicertglobalrootg2crtpem---tls-digicert-root-g2-certificate))
 - Signed by Digicert with SHA256
 - This is the `connect.nulogy.net` HTTPS/TLS certificate for our Production environment
 - For use with CNEDI and B2Bi Production
-- Note about the file name: .crt and .pem file extensions are used interchangebly
+- Note about the file name: .crt and .pem file extensions are used interchangeably
```

Comment:
You can remove this line since the newly uploaded cert has a .crt extension (instead of .pem)